app/models/post.rb

```diff
@@ -3,6 +3,10 @@ class Post < ApplicationRecord
   has_many :comments, dependent: :destroy
   has_many :likes, dependent: :destroy
 
+  validates :title, presence: true, length: { maximum: 250 }
+  validates :comments_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+  validates :likes_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+
```

Comment:
> Add unit specs for all of your models' methods and validations.

- [ ] Looking at your model specs, you don't have any for `comment and like` model. According to the requirement, you are meant to `Add unit specs for all of your models' methods and validations.`. I'll suggest you add this for likes and comments too and remember to test all the validations and model methods. :+1: :waffle: :smile: